src/Raven.Server/RavenServer.cs

```diff
@@ -1116,6 +1117,8 @@ private async Task DoActualCertificateRefresh(CertificateUtils.CertificateHolder
                     AlertType.Certificates_ReplaceError,
                     NotificationSeverity.Error,
                     details: new ExceptionDetails(e)));
+
+                throw;
```

Comment:
where this method is used? this will not kill the server?

Reply:
```
try
{
    var task = DoActualCertificateRefresh(currentCertificate, raftRequestId, forceRenew: forceRenew);
    tcs.SetResult(task);
}
catch (Exception e)
{
    tcs.SetException(e);
}
```

so we simply set the task